win32ss/include/ntuser.h

```diff
@@ -2125,11 +2125,7 @@ NtUserEnableScrollBar(
     UINT wSBflags,
     UINT wArrows);
 
-BOOL
-NTAPI
-NtUserEndDeferWindowPosEx(
-    HDWP WinPosInfo,
-    DWORD Unknown1);
+BOOL NTAPI NtUserEndDeferWindowPosEx(HDWP WinPosInfo, BOOL bAsync);
```

Comment:
Please revert, this breaks the code style used in this header.

Reply:
Reverted coding style. OK.